Error: run_as section must specify exactly one identity. Neither service_principal_name nor user_name is specified
  in databricks.yml:4:9

  "run_as": {},
